### PR TITLE
NMS-13758: Split database report docs

### PR DESCRIPTION
**Pull request by mfuhrmann**
### All Contributors

* [x] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): https://issues.opennms.org/browse/NMS-13758

### CODE REVIEW

**Review by mmahacek — CHANGES_REQUESTED** [review]
IMO, this was broken down into too many pages.  There are several pages with either a single sentence or paragraph that could still be combined so the user doesn't have to jump pack and forth to see all the info.  This section should be a max of four pages plus the introduction.

**Comment by Bonrob2** [comment]
Agree with Mark. I will make comments in the files, but also suggest the following overall structure:

1. Database Reports

2. Report management 
- Use following as intro text: This section describes how to modify existing reports and create custom reports.
- "Modify existing reports" topic
- "Add a custom report" topic

3. Jasper studio use - I think this section could be better structured to flow better. There is mention of the Measurements API, then a section on Accessing Performance Data later. We should probably create a separate Jira ticket to deal with rewriting the section. 

4. Create PDF reports from Grafana dashboards

File: docs/modules/operation/pages/database-reports/modify-report.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[[ga-database-report-modify-report]]
+== Modify existing reports
+
+All default {page-component-title} reports are located in `$OPENNMS_HOME/etc/report-templates`.
+Each .jrxml file located there can be modified; the changes are applied the next time {page-component-title} creates a report.
+
+When a subreport has been modified, {page-component-title} will detect a change based on the report's `lastModified` time and will recompile the report.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/database-reports/modify-report.adoc:7]:
```suggestion
When a subreport has been modified, {page-component-title} detects a change based on the report's `lastModified` time and recompiles the report.
```

File: docs/modules/operation/pages/database-reports/limitations.adoc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[[ga-database-reports-limitations]]
+== Limitations
+
+ * Only a JDBC Datasource to the {page-component-title} database connection can be passed to a report, or no datasource at all.
+One does not have to use the data source, though.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/database-reports/limitations.adoc:5]:
```suggestion
One does not have to use the datasource.
```

**Comment by mfuhrmann** [comment]
Closing here and starting from scratch.

**Pull request closed without merging**